daemon/sql/blockstats.go

```diff
@@ -4,6 +4,7 @@ type BlockStats struct {
 	Number       uint   `json:"number" gorm:"primaryKey;autoIncrement:false"`
 	Timestamp    uint64 `json:"timestamp"`
 	BaseFee      string `json:"baseFee"`
+	BaseFeeNext  string `gorm:"-" json:"baseFeeNext,omitempty"`
```

Comment:
nit remove? @stevengcook  this isn't needed anymore

Reply:
ah thanks!